packages/evm/src/exceptions.ts

```diff
@@ -24,8 +24,6 @@ export enum ERROR {
   INVALID_INPUT_LENGTH = 'invalid input length',
 
   AUTHCALL_UNSET = 'attempting to AUTHCALL without AUTH set',
-  AUTHCALL_NONZERO_VALUEEXT = 'attempting to execute AUTHCALL with nonzero external value',
-  AUTH_INVALID_S = 'invalid Signature: s-values greater than secp256k1n/2 are considered invalid',
 
```

Comment:
Did I miss where this invalid S value issue is removed?

Reply:
I had a discussion with lightclient on R&D and they said that this invalid s should push 0 on stack and not go out of gas.